### ISSUE TITLE
Version 2.7.0.1 Install 404 page not found

### ISSUE THREAD

**Issue by mdgann** [geoffhumphrey/brewcompetitiononlineentry#1552]
**BCOE&M Info**
Version: 2.7.0.1
Installation URL:  https://2manybeers.com/homebrew (working)
                            https://2manybeers.com/commercial (not working)
Hosted Installation? No

**Describe the Bug**
I am using Hostgator as my provider. I have an installation installed in a \homebrew subdirectory, and it works perfectly as expected.

I am now trying to install another copy in a \commercial subdirectory. This will use its own database, separate from the homebrew competition database for a commercial competition.

I have followed all the installation steps like I did before and uploaded the files. When I browse to https://2manybeers.com/commercial to start the setup, I get a 404 page does not exist message. 

This may not be the right place to post, but I don't know of another place and can't figure this out. Any help is appreciated. 

![404 error](https://github.com/user-attachments/assets/bd95f81e-b942-46a9-a0a8-ed43aea90a72)

**Comment by r-squared** [comment]
It looks like you're being redirected to the root folder, did you set the correct $sub_directory in config.php.

**Comment by mdgann** [comment]
In Config.php I have:
$sub_directory = '/commercial';

in .htaccess I have:
RewriteBase /commercial

**Comment by mdgann** [comment]
When I manually add an index.html file to the \commercial directory, I get that file just by browsing to https://2manybeers.com/commercial

Therefore I am assuming when I try it normally, index.php is being run and something it not right. Is there a way to determine what file is not being found, if that's really the case?

Our commercial competition was supposed to open today, so I'm in a desperate bind.

**Comment by r-squared** [comment]
What about the .htaccess in your root folder? You may need a "RewriteBase /" in there.

**Comment by mdgann** [comment]
I have two sub directories:
/homebrew
/commercial

I'm afraid if I add RewriteBase /commercial to the root I won't have access to /homebrew???

I tried to install an earlier version, V2.6.2.1. I was hoping if I could get that version to install, I could them upgrade to V2.7.0.1. I had more success, V2.6.2.1 went through the initial setup. However when I start editing the competition info, like the competiiton dates, it doesn;t actually save them. 

I'm not sure what I am missing. My web server should be fine, the homebrew competition is working okay,

**Comment by geoffhumphrey** [comment]
If your host provider allows it, a quicker fix may be to create two subdomains (e.g., commercial.2manybeers.com and homebrew.2manybeers.com) and point those to their respective subfolders on your server. By doing that, you can leave the $sub_folder variable in config.php blank for the commercial installation.

**Comment by mdgann** [comment]
Hi Geoff, 

I tried this. After I did it the homebrew competition install continued to work. The commercial got a little further but stopped with a 404 error with this in the address bar of the browser: https://2manybeers.com/setup.php?section=step0

I need to get this up and running. Do you think I should delete the homebrew competition install and start over with just 1 install for the commcial competition? The homebrew competition is over, so I can export the data and delete it if I have to. 

Thanks for the help.

MIke

**Comment by geoffhumphrey** [comment]
That may be your best approach since you're in a time pinch. However, there were some problems reported with the 2.7.0 setup. I believe I've got them addressed in the latest commit - since you're starting from a clean install, I think it'll be OK to try that codebase. You can get it here: https://github.com/geoffhumphrey/brewcompetitiononlineentry/archive/refs/heads/master.zip

**Comment by mdgann** [comment]
Hi Geoff - 

I deleted the previous homebrew competion install. 

I installed the commercial competition setup again, using the files you linked above. When I browsed to the site for the initial setup it failed with this error:

![V2 7 1_Setup_Error_Message](https://github.com/user-attachments/assets/1c9eda20-437a-4672-81f0-5211108020d0)

I then imported the baseline database and I was able to get the site up and running:
https://2manybeers.com/commercial

Thanks for your help!
